### DISCUSSION TITLE
disable "srflx" for some user

### DISCUSSION THREAD

**Discussion by Ali-Mirghasemi** [webrtc-rs/webrtc#63] [Q&A]
hi

i have 2 questions

1.  is possible to disable "srflx" response from TURN server for some of users?

2. can we capture or access relay data for RTCPeer or data that TURN server pass to other user?

**Comment by rainliu** [comment]
could you file issue, instead of asking question here?

**Reply by Ali-Mirghasemi** [reply]
i ask here because i'm not sure it's a issue bug
ok i'll open issue except open discuse here
